src/TimingData.h

```diff
@@ -186,7 +186,7 @@ class TimingData
 	}
 	TimingSegment* GetSegmentAtBeat( float fBeat, TimingSegmentType tst )
 	{
-		return const_cast<TimingSegment*>( GetSegmentAtBeat(fBeat, tst) );
+		return GetSegmentAtRow( BeatToNoteRow(fBeat), tst );
```

Comment:
What the... was this function just infinitely recursing before??

Reply:
I suppose so. It looks like nothing calls these functions, so it was possibly never an issue.